spec/policies/presentation_policy_spec.rb

```diff
@@ -24,16 +26,56 @@
       end
     end
 
+    context 'the person is an admin' do
+      subject { described_class.new(admin, presentation) }
+      let(:admin) { FactoryGirl.build(:person, :admin) }
+      let(:presentation) { FactoryGirl.build(:presentation) }
+
+      describe '.update?' do
+        it 'returns true' do
+          expect(subject.update?).to eq(true)
+        end
+      end
+
+      describe '.edit?' do
+        it 'calls the .update? method' do
+          expect(subject).to receive(:update?)
+          subject.edit?
+        end
+      end
+    end
+
     context 'the person is a non admin' do
-      subject { PresentationPolicy.new(non_admin, 'any_object') }
+      subject { PresentationPolicy.new(non_admin, presentation) }
       let(:non_admin) { FactoryGirl.build(:person) }
+      let(:presentation) { FactoryGirl.build(:presentation) }
+
+      describe '.update?' do
+        context %(the presentation being taken action on
+        belongs to the person) do
+
+          before do
+            non_admin.presentations << presentation
+          end
 
-      policy_methods.each do |method|
-        describe method.to_s do
           it 'returns true' do
-            expect(subject.send(method)).to eq(true)
+            expect(subject.update?).to eq(true)
           end
         end
+        context %(the presentation being taken action on
+        doesnt belong to the person) do
+
+          it 'returns false' do
+            expect(subject.update?).to eq(false)
+          end
+        end
+      end
+
+      describe '.edit?' do
+        it 'calls the .update? method' do
+          expect(subject).to receive(:update?)
```

Comment:
Not really a fan of this because the test requires internal knowlege of how `.edit?` is implemented. I know its done to save test duplication. 

But I would rather explicit tests